language/move-prover/spec-lang/src/env.rs

```diff
@@ -140,11 +153,16 @@ pub const EXPORT_ENSURES_PRAGMA: &str = "export_ensures";
 pub struct Loc {
     file_id: FileId,
     span: Span,
+    annotation: Option<String>
```

Comment:
Oops? Loc is used all over the place as map keys and elsehwere. It should have no extra field. Why is this needed? It doesn't seem to make logical sense to add data to this very basic type.